### DISCUSSION TITLE
Sum over one column?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#2437] [Q&A]
Question from a casual user: How can I build the sum over all values in one column? I tried '+', but that doesn't do the job at all.

Also, I tried to read in the documentation about `aggregate-col`, but [when I search for "aggregate-col"](https://www.visidata.org/search/?q=aggregate-col) only pages come up which do not contain "aggregate-col"... (Admittedly the first pages _does_ talk about the "+" operation.)

**Comment by saulpw** [comment]
`z+sum` will give you the result in the status (and as a memory variable).

**Reply by halloleo** [reply]
Thank you! 😄 

However, I don't get anything in the status row. After `z+sum` the status row reads for me:

```
1› MYFILE| "sum" | C z+   memo-aggregate         6 rows  •1
```

How can I read out the memory variable? I tried `Alt-M` but somehow it doesn't open the Memory sheet for me.

**Reply by anjakefala** [reply]
Hey @halloleo! What version of VisiData are you on?

`Alt+Shift+M` opens the memo sheet. =)

**Reply by halloleo** [reply]
I'm using version 2.8 (in an air-gapped environment - not easy to upgrade).

**Reply by saulpw** [reply]
@halloleo it should be there in the status messages somewhere.  Try `Ctrl+P` to see previous status messages.
